_topic_maps/_topic_map.yml

```diff
@@ -1990,11 +1990,11 @@ Topics:
   - Name: Using remote worker node at the network edge
     File: nodes-edge-remote-workers
 ---
-Name: Windows Container Support for OpenShift
+Name: Red Hat OpenShift support for Windows Containers
```

Comment:
This is changing the name of the actual book. Has this change been approved by DPM/CS, etc.? We would need to change the book name on the portal (and add redirects for each book's versions I would think). 

If it does change, please communicate this out to the team so everyone is aware too.

Reply:
Reverted this change